README.md

```diff
@@ -6,38 +6,26 @@ This library can receive and parse data from JIRA webhook, convert it into easil
 It comes loaded with a few [default listeners](#default-listeners) for the most common scenarios, but you can [extend it](#customizing) with your own custom logic too.
```

Comment:
@l80  Both links weren't working actually :( That's cause I had removed those headings(paragraphs) and moved the text somewhere else. So now I just removed the links all together.